requirements.txt

```diff
@@ -0,0 +1,2 @@
+isodate
```

Comment:
You should pin to specific versions or provide expected ranges.  Otherwise these can auto update (during some deployment) and break. Also, this makes it hard to make sure all dependencies are compatible as well.